src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -675,9 +681,35 @@ export class ResultsViewPageStore {
         invoke:()=>Promise.resolve(_.keyBy(this.studies.result, x=>x.studyId))
     }, {});
 
-    private getGermlineSampleListId(studyId:string):string {
-        return `${studyId}_germline`;
-    }
+    readonly sampleListIdsInStudies = remoteData({
+        await: () => [this.studies],
+        invoke: async () => {
+            const studies: string[] = this.studyIds;
+            return await Promise.all(studies.map(studyId => {
```

Comment:
return _.flatten(await... ) would be more intuitive I think, to return a flat list

Reply:
I did it this way to keep the current logic in `germlineConsentedSamples` intact. Seems like we are already doing the flattening there.

Reply:
Hmm, actually it looks okay to be flattened here...